x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -352,10 +352,13 @@ export const endpointMiddlewareFactory: ImmutableMiddlewareFactory<EndpointState
       });
 
       try {
+        const { page, pageSize } = getActivityLogDataPaging(getState());
```

Comment:
use paging params from the first request itself to make consistent API calls

![Screenshot 2021-06-08 at 12 31 21](https://user-images.githubusercontent.com/1849116/121170394-71166300-c855-11eb-90aa-c3fbaa23d3e9.png)